src/app/_components/GoogleLogin/GoogleLoginButton.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button } from "@mui/material";
+import React from "react";
+import GoogleIcon from "@mui/icons-material/Google";
+
+interface Props {
+  onLogin: () => void;
+}
+
+const GoogleLoginButton = ({
+  onLogin,
+}: Props): JSX.Element => {
+  const handleLogin = () => {
+    onLogin();
+  };
+
+  return (
+    <Button
+      variant="contained"
+      startIcon={<GoogleIcon />}
+      sx={{
```

Comment:
@DylanAlifu you should be able to remove all `sx` to just use the default mui button now. Rebase with dev to get in the theme changes for that tho.

Reply:
sx styling removed. Applied global styling.